source/reference/ejson.txt

```diff
@@ -24,10 +24,15 @@ to transform BSON data.
 
 .. list-table::
    :header-rows: 1
+   :widths: 30, 70
 
    * - EJSON Method
      - Use
 
+   * - :ref:`EJSON.deserialize() <mongosh-ejson-deserialize>`
+     - Convert Extended JSON objects to JavaScript objects. This method
```

Comment:
```suggestion
     - Convert Extended JSON objects to BSON objects. This method
```

We refer to these as `BSON objects` in the descriptions of the other three methods, so I think it’s worth being consistent here